ComSemApp/models.py

```diff
@@ -267,13 +274,27 @@ class Meta:
         verbose_name = "Student Attempt"
         unique_together = ("student_submission", "expression")
 
+class ReviewAttempt(models.Model):
+    expression = models.ForeignKey('Expression', on_delete=models.CASCADE)
+    course = models.ForeignKey('Course', on_delete=models.CASCADE)
+    student = models.ForeignKey('Student', on_delete=models.SET_NULL, blank=True, null=True)
+    date = models.DateTimeField(auto_now_add=True)
+    correct = models.BooleanField(default=None)
+    response_time = models.FloatField()
```

Comment:
Django has a DurationField which would be perfect here https://docs.djangoproject.com/en/2.0/ref/models/fields/#django.db.models.DurationField
Up to you if you feel it's worth changing.